### ISSUE TITLE
include iced in browser

### ISSUE THREAD

**Issue by elgubenis** [maxtaco/coffee-script#167]
hi, im compiling some coffee-script to javascript in an ace-editor, so i use the browser version of coffee-script and do CoffeeScript.compile('coffee-code'); and out comes javascript code. Now i would like to use IcedCoffeeScript to compile iced in browser to javascript, but im not sure that I use the right file to include, would it be this one? https://github.com/maxtaco/coffee-script/blob/iced2/extras/iced-coffee-script-108.0.9-min.js

**Comment by chester1000** [comment]
1. Go to https://github.com/maxtaco/coffee-script/tree/iced2/extras ,
1. Pick latest `.min` version (bottom),
1. Copy it to your project,
1. Add `<script src="js/iced-coffee-script-108.0.9-min.js"></script>` in `<head>`,
1. Add `<script type="text/coffeescript" src="main.iced"></script>` for each iced script you want to include.

**NOTE:** It's not the best idea to include original `.iced` scripts in your production code. Better just compile it and include as a `.js` file.

**tl;dr:** Yes, current version is `108.0.9`.
